tests/test_operators.py

```diff
@@ -744,34 +740,30 @@ async def test_folder_update_fails_with_bad_patch(self):
         """Test folder update raises error with improper JSON Patch."""
         patch = [{"op": "replace", "path": "/nothing"}]
         operator = FolderOperator(self.client)
-        operator.db_service.exists.return_value = futurized(True)
         operator.db_service.read.return_value = futurized(self.test_folder)
         with self.assertRaises(HTTPBadRequest):
             await operator.update_folder(self.test_folder, patch)
-            operator.db_service.exists.assert_called_once()
             operator.db_service.read.assert_called_once()
 
     async def test_folder_object_update_fails(self):
         """Test folder update fails."""
         operator = FolderOperator(self.client)
-        operator.db_service.exists.side_effect = ConnectionFailure
+        operator.db_service.update.side_effect = ConnectionFailure
         with self.assertRaises(HTTPBadRequest):
             await operator.update_folder(self.test_folder, [])
 
     async def test_folder_object_remove_passes(self):
         """Test remove object method for folders works."""
         operator = FolderOperator(self.client)
-        operator.db_service.exists.return_value = futurized(True)
         operator.db_service.remove.return_value = futurized(self.test_folder)
         await operator.remove_object(self.test_folder, "study", self.accession_id)
-        operator.db_service.exists.assert_called_once()
         operator.db_service.remove.assert_called_once()
         self.assertEqual(len(operator.db_service.remove.mock_calls), 1)
 
     async def test_folder_object_remove_fails(self):
         """Test folder remove object fails."""
         operator = FolderOperator(self.client)
-        operator.db_service.exists.side_effect = ConnectionFailure
+        operator.db_service.remove.side_effect = ConnectionFailure
         with self.assertRaises(HTTPBadRequest):
             await operator.remove_object(self.test_folder, "study", self.accession_id)
 
```

Comment:
Just a general comment: test, test2, patch, patch2 etc are not the most descriptive for variables or values

Reply:
variable names and function descriptions improved, so that they make more sense now.